sentry-core/src/main/java/io/sentry/core/SentryClient.java

```diff
@@ -26,58 +20,27 @@ public boolean isEnabled() {
     return isEnabled;
   }
 
-  public SentryClient(SentryOptions options) {
+  public SentryClient(SentryOptions options, @Nullable AsyncConnection connection) {
```

Comment:
would not it be better just to have an overload instead of forcing to pass null if we want to?

Reply:
Great! yes now that I know I can have default implementation :)
Thanks

Reply:
Right! That's a good point.
Thanks